tests/at_tests.cpp

```diff
@@ -26,39 +26,39 @@ SUITE(at_tests)
 {
     TEST(static_array)
     {
-        int a[] = { 1, 2, 3, 4 };
+        size_t a[] = { 1, 2, 3, 4 };
 
-        for (int i = 0; i < 4; ++i)
+        for (size_t i = 0; i < 4; ++i)
             CHECK(at(a, i) == i+1);
```

Comment:
You tell me 😄 

That at if the one that's been there for a while (this is testing `gsl::at` and not the new `gsl::span::at`).